index.html

```diff
@@ -0,0 +1,240 @@
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="menu.css">
+    <title>Document</title>
+</head>
+  <body>
+    
+    <nav class="toolbar">
+      <div class="logo"><a href="">My Logo</a></div>
+      <div class="menu_main">
+        <input type="checkbox" class="toggler">
+        <div class="humbuger"><div></div></div>
+        <div class="menu">
+          <div class="overlay">
+            <div>
+              <ul>
+                <li><a href="">Portifolio</a></li>
+                <li><a href="">About</a></li>
+                <li><a href="">Context</a></li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+
+    </nav>
+    <section class="headline">
+      <div class="container">
+        <h1>I’m Patrick
+            glad to see you!</h1>
+        <p>I’m a software developer! 
+            I can help you build a product ,
+            feature or website Look through 
+            some of my work and experience! 
+            If you like what you see and have 
+            a project you need coded, 
+            don’t hestiate to contact me.</p>
+        <h3>
+            <a href="">LET’S CONNECT</a> 
+        </h3>  
+        <ul class="social-media">
+            <li><a href="twiter"><img src="../Portfolio-finish-mobile-version/img/twiter.svg" alt=""></a></li>
+            <li><a href="linkedin"><img src="../Portfolio-finish-mobile-version/img/Linkedin.svg" alt=""></a></li>
+            <li><a href="medium"><img src="../Portfolio-finish-mobile-version/img/medium.svg" alt=""></a></li>
+            <li><a href="github"><img src="../Portfolio-finish-mobile-version/img/github.svg" alt=""></a></li>
+            <li><a href="chill"><img src="../Portfolio-finish-mobile-version/img/chill.svg" alt=""></a></li>
+        </ul> 
+      </div>
+    </section>
+    <main class="work">
+      <ul>
+      <li class="item tonic">
+        <img src="../Portfolio-finish-mobile-version/img/work1.svg" alt="">
+        <div>
+          <div>
+            <h1>Tonic</h1>
+            <ul class="canopy">
+              <li><h3>CANOPY</h3></li>
+              <li><img src="../Portfolio-finish-mobile-version/img/circle.svg" alt=""></li>
+              <li><p>Back End Dev</p></li>
+              <li><img src="../Portfolio-finish-mobile-version/img/circle.svg" alt=""></li>
+              <li><p>2015</p></li>
+            </ul>
+            </div>
+            <p>
+                A daily selection of privately
+                personalized reads; no accounts
+                or sign-ups required.
+            </p>
+            <ul class="languages">
+              <li><a href="">html</a></li>
+              <li><a href="">css</a></li>
+              <li><a href="">javascript</a></li>
+              <li><a href="">Ruby on rails</a></li>
+            </ul>
+            <div class="rev">
+              <button><a href="" class="btn">See Project</a></button>
```

Comment:
- Let's also add the missing interaction between buttons by indicating when it's active, hovered, pressed, etc. kindly check the Figma design to see the correct color used to improve our design.